servicex/rabbit_adaptor.py

```diff
@@ -43,17 +44,17 @@ def __init__(self, amqp_url):
         """
         self._connection = None
         self._channel = None
-        self._url = amqp_url
+        self._url = [pika.URLParameters(u) for u in amqp_url.split(",")]
```

Comment:
Is this still a _url? Should we rename it to someone else now? It is a class of somesort, if I understand the docs. Perhaps for future readability we should not name it _url?

Reply:
Good suggestion. Changed to _url_list